config/routes.rb

```diff
@@ -56,6 +56,10 @@
 
     resources :remote_phone_call_events, only: %i[index show]
     resources :users, except: %i[new create]
+    resources :sensors do
+      resources :sensor_rules
```

Comment:
here you only want nested routes for :new, :create, :index

:show, :destroy, :edit and :update don't need to be nested